model/histogram/float_histogram.go

```diff
@@ -334,6 +334,33 @@ func (h *FloatHistogram) Equals(h2 *FloatHistogram) bool {
 	return true
 }
 
+// Size returns the size of the whole fields histogram in bytes.
+// NOTE: this is only valid for 64 bit architectures.
+func (fh *FloatHistogram) Size() int {
+	// Size of each slice separately
+	posSpanSize := len(fh.PositiveSpans) * 8     // 8 bytes (int32 + uint32)
+	negSpanSize := len(fh.NegativeSpans) * 8     // 8 bytes (int32 + uint32)
+	posBucketSize := len(fh.PositiveBuckets) * 8 // 8 bytes (float64)
+	negBucketSize := len(fh.NegativeBuckets) * 8 // 8 bytes (float64)
+
+	// Total size of the struct
+
+	// fh is 8 bytes
+	// fh.CounterResetHint is 1 byte
+	// fh.Schema is 4 bytes
+	// fh.ZeroThreshold is 8 bytes
+	// fh.ZeroCount is 8 bytes
+	// fh.Count is 8 bytes
+	// fh.Sum is 8 bytes
+	// fh.PositiveSpans is 24 bytes
+	// fh.NegativeSpans is 24 bytes
+	// fh.PositiveBuckets is 24 bytes
+	// fh.NegativeBuckets is 24 bytes
+	structSize := 141
```

Comment:
So this should be 144. (I suggested 136 in the last round of review, but I was not aware that your intention is to include the size of the pointer to FloatHistogram).